sorc/checkout.sh

```diff
@@ -31,11 +27,7 @@ if [[ ! -d fv3gfs.fd ]] ; then
     rm -f ${topdir}/checkout-fv3gfs.log
     git clone https://github.com/ufs-community/ufs-weather-model fv3gfs.fd >> ${topdir}/checkout-fv3gfs.log 2>&1
     cd fv3gfs.fd
-    if [ ${run_ccpp:-"NO"} = "NO" ]; then
-     git checkout GFS.v16.0.16
-    else
-     git checkout 9350745855aebe0790813e0ed2ba5ad680e3f75c
-    fi
+    git checkout 9350745855aebe0790813e0ed2ba5ad680e3f75c
```

Comment:
Would prefer a tag here, if there is one.

Reply:
I prefer tags as well but there aren't any recent ones. Will definitely retest and move to a new tag when one is made.